packages/qulacs/tests/qulacs/circuit/test_convert_circuit.py

```diff
@@ -155,21 +165,25 @@ def test_convert_u_gate() -> None:
 def test_convert_pauli_gate() -> None:
     g = gates.Pauli((11, 7, 13), (2, 3, 1))
     converted = convert_gate(g)
-    expected = qulacs.gate.Pauli((11, 7, 13), (2, 3, 1))
+    expected = qulacs.gate.Pauli(cast_to_list((11, 7, 13)), cast_to_list((2, 3, 1)))
     assert gates_equal(converted, expected)
 
 
 def test_convert_pauli_rotation_gate() -> None:
     g = gates.PauliRotation((11, 7, 13), (2, 3, 1), 0.125)
     converted = convert_gate(g)
-    expected = qulacs.gate.PauliRotation((11, 7, 13), (2, 3, 1), -0.125)
+    expected = qulacs.gate.PauliRotation(
+        cast_to_list((11, 7, 13)), cast_to_list((2, 3, 1)), -0.125
+    )
     assert gates_equal(converted, expected)
 
     # Check rotation angle sign
     c = np.cos(np.pi / 4)
     s = np.sin(np.pi / 4)
+    # the `.get_matrix()` method returns NDArray[complex128, _Shape]
+    # (_Shape should be the first argument)
```

Comment:
I think this is clearer:
```suggestion
    # We need to disable type check due to an error in qulacs type annotation
    # https://github.com/qulacs/qulacs/issues/537
```